### PR TITLE
Change sidekiq to delayed job

### PR DESCRIPTION
**Pull request by laurajaime**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -48,7 +48,7 @@ group :development, :staging do
 end
 
 group :production, :staging do
-  gem "sidekiq", "~>6.3.1"
+  gem "delayed_job_active_record"
```
Comment (tramuntanal) [inline Gemfile:51]:
```suggestion
  gem "daemons"
  gem "delayed_job_active_record"
```

and re install

**Review by tramuntanal — CHANGES_REQUESTED** [review]
delayed job needs the "daemons" gem. You will have to `bundle install` again

File: Gemfile (hunk 1 above)
Comment (laurajaime) [reply Gemfile:52]: done!

**Review by tramuntanal — APPROVED** [review]
The `db/schema.rb` diff is for everything, very strange.
I approve, but can you check why all tables are marked as different?

**Pull request merged**